docker-compose-esignet/config/esignet-default.properties

```diff
@@ -0,0 +1,577 @@
+  \ 'response_modes_supported' : { 'query' }, \
+  \ 'token_endpoint_auth_methods_supported' : ${mosip.esignet.supported.client.auth.methods}, \
+  \ 'token_endpoint_auth_signing_alg_values_supported' : {'RS256'}, \
+  \ 'userinfo_signing_alg_values_supported' : {'RS256'}, \
+  \ 'userinfo_encryption_alg_values_supported' : {'RSAXXXXX'},\
+  \ 'userinfo_encryption_enc_values_supported' : {'A128GCM'}, \
+  \ 'id_token_signing_alg_values_supported' : {'RS256'}, \
+  \ 'claim_types_supported': {'normal'}, \
+  \ 'claims_parameter_supported' : true, \
+  \ 'display_values_supported' : ${mosip.esignet.supported.ui.displays}, \
+  \ 'subject_types_supported' : { 'pairwise' }, \
+  \ 'claims_supported' : {'name','address','gender','birthdate','picture','email','phone_number','individual_id'}, \
+  \ 'acr_values_supported' : {'mosip:idp:acr:static-code', 'mosip:idp:acr:generated-code', 'mosip:idp:acr:linked-wallet', 'mosip:idp:acr:biometrics'},\
+  \ 'request_parameter_supported' : false }
+
+##----------------------------------------- Database properties --------------------------------------------------------
+
+mosip.esignet.database.hostname=database
+mosip.esignet.database.port=5432
+spring.datasource.url=jdbc:postgresql://${mosip.esignet.database.hostname}:${mosip.esignet.database.port}/mosip_esignet?currentSchema=esignet
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL95Dialect
+spring.jpa.show-sql=false
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+
+#------------------------------------ Key-manager specific properties --------------------------------------------------
+#Crypto asymmetric algorithm name
+mosip.kernel.crypto.asymmetric-algorithm-name=RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING
+#Crypto symmetric algorithm name
+mosip.kernel.crypto.symmetric-algorithm-name=AES/GCM/PKCS5Padding
+#Keygenerator asymmetric algorithm name
+mosip.kernel.keygenerator.asymmetric-algorithm-name=RSA
+#Keygenerator symmetric algorithm name
+mosip.kernel.keygenerator.symmetric-algorithm-name=AES
+#Asymmetric algorithm key length
+mosip.kernel.keygenerator.asymmetric-key-length=2048
+#Symmetric algorithm key length
+mosip.kernel.keygenerator.symmetric-key-length=256
+#Encrypted data and encrypted symmetric key separator
+mosip.kernel.data-key-splitter=#KEY_SPLITTER#
+#GCM tag length
+mosip.kernel.crypto.gcm-tag-length=128
+#Hash algo name
+mosip.kernel.crypto.hash-algorithm-name=PBKDF2WithHmacSHA512
+#Symmtric key length used in hash
+mosip.kernel.crypto.hash-symmetric-key-length=256
+#No of iterations in hash
+mosip.kernel.crypto.hash-iteration=100000
+#Sign algo name
+mosip.kernel.crypto.sign-algorithm-name=RS256
+#Certificate Sign algo name
+mosip.kernel.certificate.sign.algorithm=SHA256withRSA
+
+mosip.kernel.keymanager.hsm.config-path=local.p12
```

Comment:
how are we initializing the keys in this p12 ?
Also is this file mounted in volume to withstand the docker compose restarts ?

Reply:
we are not generating p12 file, local setup is working with the docker image

Reply:
Is this file mounted in volume to withstand the docker compose restarts ?

Reply:
added volume bind for this file